src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+
+<div class="how-to">
+  <strong>How To: File Handling</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/experimental-features/#file-handling" target="_blank" rel="noopener">this documentation</a> to learn more!
+  </p>
+</div>
+
+## Shortcuts
+
+Using the Web Shortcuts API, PWAs can tie into the shortcuts UI in Windows, enabling users to jump to specific pages or actions in your PWA! This can be useful for
+
+- Mail Applications for example, as you can let users jump straight to writing a new email from a right click on your App Icon in the start menu and taskbar!
+- Social Media Apps such as Twitter, which enables users to jump straight into creating a tweet, the notifications screen, reading their DMs and more, all from a right click of the Twitter icon.
+
+<img src="/docs/windows/pwa-api-documentation/shortcuts.png" width="350px" alt="A screenshot of what the shorcuts UI looks like on Windows" /> 
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/webappmanifests#use-shortcuts-to-provide-quick-access-to-features" target="_blank" rel="noopener">this tutorial</a> to learn how to add shortcuts to your PWA.
+  </p>
+</div>
+
+## The Clipboard
+
+Want to be able to programmatically copy or paste content to and from your PWA? While this was possible before with an older clipboard API built into browsers, the experience was not great. With the new Async Clipboard API it is much easier to programmatically copy and paste everything from text to files. This also works seamlessly with the Cloud Clipboard in Windows!
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a demo <a href="https://wiggly-impossible-crayfish.glitch.me/clipboard.html" target="_blank" rel="noopener">here</a> of the Clipboard API!
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://web.dev/async-clipboard/" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Async Clipboard API and enable this functionality in your PWA.
+  </p>
+</div>
+
+## Offline
+
+Progressive Web Apps, unlike traditional web apps, can not only work great offline, but with the APIs available in Edge and other Chromium based browsers your app can provide specific tailored experiences for offline use. Service Workers are the key technology that enables this and therefore you will need one before you can use these APIs (Use [PWABuilder](https://www.pwabuilder.com) to easily grab one, or check out our recommendations [here](/posts/building-pwas-with-web-components!/)). Let’s go through a few of these APIs:
```

Comment:
Maybe reorder the wording here? "Unlike traditional web apps, PWAs can not only work great offline..."

Reply:
Period after use these APIs. And I think the parens are unnecessary.